client/components/payment-status/payment-status.js

```diff
@@ -109,7 +109,7 @@ export default class PaymentStatus extends React.Component {
               <div className="row">
                 <Link
                   className="button full"
-                  to={`/${orgSlug}/status`}
+                  to={`/${orgSlug}/payment/process`}
```

Comment:
I think the setting which controls the behavior requested in the previous comment here is not there yet, right?

BTW I have observed that if I go to the status component, the captive portal login is triggered.
I think we need a dedicated setting which controls whether captive portal login is done for unverified users having bank_card as registration method.

In the use case we're working, we'll set this setting to false so that cp login is not done.